test/test_prototype_transforms_functional.py

```diff
@@ -199,21 +199,38 @@ def resize_bounding_box():
             yield SampleInput(bounding_box, size=size, image_size=bounding_box.image_size)
 
 
-class TestKernelsCommon:
-    @pytest.mark.parametrize("functional_info", FUNCTIONAL_INFOS, ids=lambda functional_info: functional_info.name)
-    def test_scriptable(self, functional_info):
-        jit.script(functional_info.functional)
-
-    @pytest.mark.parametrize(
-        ("functional_info", "sample_input"),
-        [
-            pytest.param(functional_info, sample_input, id=f"{functional_info.name}-{idx}")
-            for functional_info in FUNCTIONAL_INFOS
-            for idx, sample_input in enumerate(functional_info.sample_inputs())
-        ],
-    )
-    def test_eager_vs_scripted(self, functional_info, sample_input):
```

Comment:
Sorry, I probably miss the context but why can't we keep the `test_eager_vs_scripted` for the image cases?

Reply:
Damn I missed this. Sorry.